driver-simulator/README.md

```diff
@@ -0,0 +1,18 @@
+# Driver simulator
+
+Use this simulator to test your solution for the full-stack code challenge.
+
+The driver simulator will:
+1. Register all vehicles listed in the file `vehicles.json`
+2. Update the vehicle location until it reaches its destination
+3. De-register all vehicles when they are finished
+
+## Get started
+
+Install [node](https://github.com/creationix/nvm#install-script) and then run the following command:
```

Comment:
It would be good to specify the desired Node version, and perhaps include a `.nvmrc`.

Reply:
👍